controllers/bounce_processes.go

```diff
@@ -60,7 +60,7 @@ func (b BounceProcesses) Reconcile(r *FoundationDBClusterReconciler, context ctx
 		}
 	}
 
-	processesToBounce := fdbtypes.FilterByCondition(cluster.Status.ProcessGroups, fdbtypes.IncorrectCommandLine, true)
+	processesToBounce := fdbtypes.FilterByCondition(cluster.Status.ProcessGroups, fdbtypes.IncorrectCommandLine, true, true)
```

Comment:
Is there a good reason to change the method signature here and not use `SkipProcessGroup` in the for loop below? I would prefer that instead of changing the method since `SkipProcessGroup` also checks if a Pod is in the Pending state for a given time.

Reply:
Completely agree on this, let me make the changes.